starter-dev-e2e/cypress/integration/profile.spec.ts

```diff
@@ -0,0 +1,121 @@
+import { View } from "../utils/view";
+
+describe("Profile Page", () => {
+  beforeEach(() => {
+    cy.interceptGraphQLCalls(View.Profile);
+    cy.interceptRestCalls(View.Profile);
+
+    cy.visit("/thisdot");
+  });
+
+  it("should display correct name and username", () => {
+    cy.get(`[data-testid="profile page name"]`).should("be.visible");
+    cy.get(`[data-testid="profile page username"]`).should("be.visible");
+  });
+
+  it("should display same profile and header avatar", () => {
+    cy.wait("@UserProfileQuery");
+    cy.get(`[data-testid="user avatar header"]`).should("be.visible");
+    cy.get(
+      `[data-testid="profile page avatar"], [data-testid="user avatar header"]`
+    )
+      .should("have.attr", "src")
+      .and(
+        "match",
+        /.*(https).*(avatars\.githubusercontent\.com).*(22839396).*/
+      );
+  });
+
+  it("should display orgs in profile", () => {
+    cy.get(`[data-testid="profile page orgs"]`).should("have.length", 1);
+  });
+
+  it("should display correct count", () => {
+    cy.get(`[data-testid="profile followers count"]`).contains("10");
+    cy.get(`[data-testid="profile following count"]`).contains("12");
+    cy.get(`[data-testid="profile starred count"]`).contains("5");
+  });
+
+  it("should display correct repos", () => {
+    cy.wait("@UserReposQuery");
+    cy.get(`[data-testid="profile repo list heading"]`)
+      .should("have.length", 6)
+      .and(($h3) => {
+        expect($h3.get(0).textContent, "first item").to.contain("sentry");
+        expect($h3.get(5).textContent, "last item").to.contain(
+          "javascript-marathon-vue-js"
+        );
```

Comment:
I just sort of worry about checking for an `h3` specifically here -- maybe not all the showcases decide to use an h3 element for this.

The good news is that because of the repository tests, I have an ID set up for the repository name element/link specifically so we could check for that like so:

```
cy.wait("@UserReposQuery")
      .get(`[data-testid="profile repo list heading"]`)
      .should("have.length", 6)
      .get(`[data-testid="repository name"]`)
      .and(($a) => {
        expect($a.get(0).textContent, "first item").to.contain("sentry");
        expect($a.get(5).textContent, "last item").to.contain(
          "javascript-marathon-vue-js"
        );
      });
```

It still potentially has the problem of expecting a `$a` element, but I think all showcases are more likely guaranteed to use that instead of an `h3`

Reply:
Good catch and definitely correct when it comes to other showcases, will update in my next commit.